### PR TITLE
[docker] postgres healthcheck not operational

### PR DESCRIPTION
**Pull request by jcpsantiago**
📓 Description

_Summary of the change and link to any relevant tickets. New aliases should include details of why they are valuable_

Fixed some typos and YAML white space problems preventing the config from being valid.

I couldn't `docker-compose up` with the `name` attribute present, so I removed it in this PR:
```
(root) Additional property name is not allowed
```
But reading the [docs](https://docs.docker.com/compose/compose-file/04-version-and-name/#name-top-level-element) it sounds like it should be correct? No idea.

:octocat Type of change

_Please tick `x` relevant options, delete those not relevant_

- [ ] New feature
- [ ] Deprecate feature
- [ ] Development workflow
- [x] Documentation
- [ ] Continuous integration workflow

:beetle How Has This Been Tested?

- [ ] unit test
- [ ] linter check
- [x] GitHub Action checkers

:eyes Checklist

- [x] Code follows the [Practicalli cljstyle configuration](https://practical.li/clojure/clojure-cli/clojure-style/#cljstyle)
- [x] Add / update alias docs and README where relevant
- [x] Changelog entry describing notable changes
- [x] Request practicalli-john review the PR

### CODE REVIEW

File: resources/practicalli/application/root/compose.yaml [hunk 1]
```diff
@@ -76,7 +74,7 @@ services:
   #   environment:
   #     MYSQL_USER: "$DOCKER_SERVICE_DB_USER"
   #     MYSQL_PASSWORD: "$DOCKER_SERVICE_DB_PASSWORD"
-  #     MYSQL_DATABASE: "service"
+  #     MYSQL_DATABASE: "$DOCKER_SERVICE_DB_DB"
```
Comment (jcpsantiago) [inline resources/practicalli/application/root/compose.yaml:77]: Maybe looks nicer if they're consistent with postgres above e.g. `$DOCKER_MYSQL_DB`?

**Comment by jcpsantiago** [comment]
Postgres's healthcheck is failing with this error https://github.com/peter-evans/docker-compose-healthcheck/issues/16
The proposed solution in https://github.com/peter-evans/docker-compose-healthcheck/pull/17 is to change the command to 
```sh
["CMD-SHELL", "pg_isready -d {db} -U {user}"]
```
but it didn't work for me

**Comment by practicalli-john** [comment]
> I couldn't `docker-compose up` with the `name` attribute present, so I removed it in this PR:
> 
> ```
> (root) Additional property name is not allowed
> ```
> 
> But reading the [docs](https://docs.docker.com/compose/compose-file/04-version-and-name/#name-top-level-element) it sounds like it should be correct? No idea.

What version of Docker is being used, it may be using an older docker compose v1 spec

Also the command to run docker compose should be

```shell
docker compose up
```
rather than the older command form docker-compose up (I'll check if this is documented)

**Comment by practicalli-john** [comment]
> Postgres's healthcheck is failing with this error [peter-evans/docker-compose-healthcheck#16](https://github.com/peter-evans/docker-compose-healthcheck/issues/16) The proposed solution in [peter-evans/docker-compose-healthcheck#17](https://github.com/peter-evans/docker-compose-healthcheck/pull/17) is to change the command to
> 
> ```shell
> ["CMD-SHELL", "pg_isready -d {db} -U {user}"]
> ```
> 
> but it didn't work for me

There is a healthcheck both in `compose.yaml` and the `Dockerfile`, so some testing of both seems to be required.

**Comment by jcpsantiago** [comment]
> > I couldn't `docker-compose up` with the `name` attribute present, so I removed it in this PR:
> > ```
> > (root) Additional property name is not allowed
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > But reading the [docs](https://docs.docker.com/compose/compose-file/04-version-and-name/#name-top-level-element) it sounds like it should be correct? No idea.
> 
> I am unable to reproduce this issue, using Docker Desktop 4.18.0 (104112) on Linux
> 
> What version of Docker is being used, it may be using an older docker compose v1 spec
> 
> Also the command to run docker compose should be
> 
> ```shell
> docker compose up
> ```
> 
> rather than the older command form `docker-compose up` (I'll check if this is documented)

my bad. I'm coding on my 2012 iMac with MacOS 10, so I had an ancient docker version. I'm running 4.15 (the most I can on this beast) and now `name:` works fine, and also the healthcheck (with my changes adding `-d` and `-U`).

I'm using `docker compose up`.

**Pull request closed without merging**